test/token/ERC20/extensions/ERC4626.test.js

```diff
@@ -44,6 +45,81 @@ contract('ERC4626', function (accounts) {
     }
   });
 
+  describe('reentrancy', async function () {
+    const reenterType = {
+      Before: 1,
+      After: 2,
+    };
```

Comment:
we have an helper for that in `test/helpers/enums.js`. Doesn't make a big difference, but for consistency it might be better to use it.

```suggestion
    const reenterType = Enum('No', 'Before', 'After');
```